documentation/src/components/layout.ts

```diff
@@ -95,4 +95,9 @@ export class LayoutElement extends RouteComponent {
             </sp-theme>
         `;
     }
+
+    connectedCallback() {
```

Comment:
Realized I had added this to ALL of the extensions of this base class, so refactored.